unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Tween/ECSTweenHandler.cs

```diff
@@ -0,0 +1,184 @@
+            return false;
+
+        switch (modelA.ModeCase)
+        {
+            case PBTween.ModeOneofCase.Scale:
+                return modelB.Scale.Start.Equals(modelA.Scale.Start)
+                       && modelB.Scale.End.Equals(modelA.Scale.End);
+            case PBTween.ModeOneofCase.Rotate:
+                return modelB.Rotate.Start.Equals(modelA.Rotate.Start)
+                       && modelB.Rotate.End.Equals(modelA.Rotate.End);
+            case PBTween.ModeOneofCase.Move:
+            default:
+                return modelB.Move.Start.Equals(modelA.Move.Start)
+                       && modelB.Move.End.Equals(modelA.Move.End);
+        }
+    }
+
+    private void InitializeEasingFunctionsMap()
+    {
+        easingFunctionsMap[EasingFunction.TfEaseinsine] = Ease.InSine;
+        easingFunctionsMap[EasingFunction.TfEaseoutsine] = Ease.OutSine;
+        easingFunctionsMap[EasingFunction.TfEasesine] = Ease.InOutSine;
+        easingFunctionsMap[EasingFunction.TfEaseinquad] = Ease.InQuad;
+        easingFunctionsMap[EasingFunction.TfEaseoutquad] = Ease.OutQuad;
+        easingFunctionsMap[EasingFunction.TfEasequad] = Ease.InOutQuad;
+        easingFunctionsMap[EasingFunction.TfEaseinexpo] = Ease.InExpo;
+        easingFunctionsMap[EasingFunction.TfEaseoutexpo] = Ease.OutExpo;
+        easingFunctionsMap[EasingFunction.TfEaseexpo] = Ease.InOutExpo;
+        easingFunctionsMap[EasingFunction.TfEaseinelastic] = Ease.InElastic;
+        easingFunctionsMap[EasingFunction.TfEaseoutelastic] = Ease.OutElastic;
+        easingFunctionsMap[EasingFunction.TfEaseelastic] = Ease.InOutElastic;
+        easingFunctionsMap[EasingFunction.TfEaseinbounce] = Ease.InBounce;
+        easingFunctionsMap[EasingFunction.TfEaseoutbounce] = Ease.OutBounce;
+        easingFunctionsMap[EasingFunction.TfEasebounce] = Ease.InOutBounce;
+        easingFunctionsMap[EasingFunction.TfEaseincubic] = Ease.InCubic;
+        easingFunctionsMap[EasingFunction.TfEaseoutcubic] = Ease.OutCubic;
+        easingFunctionsMap[EasingFunction.TfEasecubic] = Ease.InOutCubic;
+        easingFunctionsMap[EasingFunction.TfEaseinquart] = Ease.InQuart;
+        easingFunctionsMap[EasingFunction.TfEaseoutquart] = Ease.OutQuart;
+        easingFunctionsMap[EasingFunction.TfEasequart] = Ease.InOutQuart;
+        easingFunctionsMap[EasingFunction.TfEaseinquint] = Ease.InQuint;
+        easingFunctionsMap[EasingFunction.TfEaseoutquint] = Ease.OutQuint;
+        easingFunctionsMap[EasingFunction.TfEasequint] = Ease.InOutQuint;
+        easingFunctionsMap[EasingFunction.TfEaseincirc] = Ease.InCirc;
+        easingFunctionsMap[EasingFunction.TfEaseoutcirc] = Ease.OutCirc;
+        easingFunctionsMap[EasingFunction.TfEasecirc] = Ease.InOutCirc;
+        easingFunctionsMap[EasingFunction.TfEaseinback] = Ease.InBack;
+        easingFunctionsMap[EasingFunction.TfEaseoutback] = Ease.OutBack;
+        easingFunctionsMap[EasingFunction.TfEaseback] = Ease.InOutBack;
+    }
```

Comment:
I think @Kinerius idea was to have it directly in the variable declaration, which together with `using static` can be a quite readable map
 
```
using static DCL.ECSComponents.EasingFunction;
using static DG.Tweening.Ease;

private readonly Dictionary<EasingFunction, Ease> easingFunctionsMap = new()
    {
        [TfEaseinsine] = InSine,
        [TfEaseoutsine] = OutSine,
        [TfEasesine] = InOutSine,
        [TfEaseinquad] = InQuad,
        [TfEaseoutquad] = OutQuad,
        [TfEasequad] = InOutQuad,
        [TfEaseinexpo] = InExpo,
        [TfEaseoutexpo] = OutExpo,
        [TfEaseexpo] = InOutExpo,
        [TfEaseinelastic] = InElastic,
        [TfEaseoutelastic] = OutElastic,
        [TfEaseelastic] = InOutElastic,
        [TfEaseinbounce] = InBounce,
        [TfEaseoutbounce] = OutBounce,
        [TfEasebounce] = InOutBounce,
        [TfEaseincubic] = InCubic,
        [TfEaseoutcubic] = OutCubic,
        [TfEasecubic] = InOutCubic,
        [TfEaseinquart] = InQuart,
        [TfEaseoutquart] = OutQuart,
        [TfEasequart] = InOutQuart,
        [TfEaseinquint] = InQuint,
        [TfEaseoutquint] = OutQuint,
        [TfEasequint] = InOutQuint,
        [TfEaseincirc] = InCirc,
        [TfEaseoutcirc] = OutCirc,
        [TfEasecirc] = InOutCirc,
        [TfEaseinback] = InBack,
        [TfEaseoutback] = OutBack,
        [TfEaseback] = InOutBack,
    };
```

Reply:
Cool thanks! now I implemented it that way.